scripts/devservices-healthcheck.sh

```diff
@@ -13,21 +13,47 @@
 #       specify distinct retries and other parameters.
 #       would also be nice to have a single ^C work.
 
-try=1
-to=3
-start=0
-delay=5
-
-sleep "$start"
-while (( $try <= $to )); do
-    echo "Checking health of postgres (try ${try} of ${to})..."
-    if docker exec sentry_postgres pg_isready -U postgres; then
-        break
-    fi
-    if (( $try == $to )); then
-        echo "Exceeded retries."
-        exit 1
-    fi
-    try=$(($try + 1))
-    sleep "$delay"
-done
+
+# Accepts following positional arguments:
+# $1 - how many times to try
+# $2 - delay between tries
+# $3 - the service (docker container) name
+# $4 - command to run inside the docker container
+function health_check {
+  local try=1
+  local to=$1
+  local start=0
+  local delay=$2
+  local service="$3"
+  local cmd="$4"
+
+
+  sleep "$start"
+  echo "Running '$cmd' in '$service'"
+  while (( $try <= $to )); do
+      echo "Checking '$service' (try ${try} of ${to})..."
+      if docker exec $service $cmd; then
+          return 0
+      fi
+
+      if (( $try == $to )); then
+          echo "Exceeded retries for '$service'"
+          return 1
+      fi
+      try=$(($try + 1))
+      sleep "$delay"
+  done
+}
```

Comment:
You might be right. But this change is still very very small and just wraps the existing code in the function to avoid code duplication. It it were something more complex it would agree that it'd have to be re-written in python.

Reply:
the original patch was small -- this is a now a rewrite and our policy for dev scripts is if there's a rewrite it is done in python so we can have linting, types, tests, etc.